3ds Max/Max2Babylon/Exporter/BabylonExporter.GLTFExporter.Light.cs

```diff
@@ -0,0 +1,92 @@
+﻿using BabylonExport.Entities;
+using GLTFExport.Entities;
+using System.Collections.Generic;
+
+namespace Max2Babylon
+{
+    internal partial class BabylonExporter
+    {
+        public const string KHR_lights = "KHR_lights";  // Name of the extension
+
+
+        /// <summary>
+        /// Add the light the global extensions
+        /// </summary>
+        /// <param name="gltf">The gltf data</param>
+        /// <param name="babylonLight">The light to export</param>
+        /// <returns>the index of the light</returns>
+        private int AddLightExtension(ref GLTF gltf, BabylonLight babylonLight)
+        {
+            if (gltf.extensionsUsed.Contains(KHR_lights) == false)
+            {
+                gltf.extensionsUsed.Add(KHR_lights);
+            }
+
+            // new light in the gltf extensions
+            GLTFLight light = new GLTFLight
+            {
+                color = babylonLight.diffuse,
+                type = ((GLTFLight.LightType)babylonLight.type).ToString(),
+                intensity = babylonLight.intensity,
+            };
+
+            switch (babylonLight.type)
+            {
+                case (0): // point
+                    light.type = GLTFLight.LightType.point.ToString();
+                    light.range = babylonLight.range;
+                    break;
+                case (1): // directional
+                    light.type = GLTFLight.LightType.directional.ToString();
+                    break;
+                case (2): // spot
+                    light.type = GLTFLight.LightType.spot.ToString();
+                    light.range = babylonLight.range;
+                    light.spot = new GLTFLight.Spot
+                    {
+                        //innerConeAngle = 0, Babylon doesn't support the innerConeAngle
```

Comment:
Is it Confirmed? I think it should be supported now

Reply:
OK I checked this is good:)

Reply:
In the loader, this parameter is not handled. And I didn't find in the documentation the equivalent.